src/pybind/mgr/dashboard/README.rst

```diff
@@ -55,6 +55,16 @@ See the `Ceph Dashboard plugin documentation
 and configure the dashboard manually and how to configure other settings, e.g.
 access to the Ceph object gateway.
 
+Supported browsers
+------------------
+
+We support `Chrome 68+ <https://www.google.com/chrome/>`_ and
+`Firefox 61+ <http://www.mozilla.org/firefox/>`_, as both these browsers are
+frequently used for testing and development.
+
+While not officially supported, other browsers like Edge 42+, Opera 41+ and
```

Comment:
Dito here about "official support" - I'd rather state that we don't explicitly test on these browsers, but they should work in theory (and that feedback is welcome, if issues are observed).